testsuite/features/support/namespaces/system.rb

```diff
@@ -128,7 +128,7 @@ def schedule_script_run(server, uid, gid, timeout, script, date)
   end
 
   ##
-  # Creates a system record in the Satellite 6 server.
+  # Creates a system record on the Spacewalk server.
```

Comment:
this could be renamed to SUMA server, spacewalk server was the super old name from RH. But better than satellite for sure

Reply:
Fixed. Thank you.